merge-specs

```diff
@@ -97,7 +97,7 @@ function loadSpec(string $path): array {
 
 function rewriteRefs(array $spec): array {
 	$readableAppID = Helpers::generateReadableAppID($spec["info"]["title"]);
-	array_walk_recursive($spec, function (string &$item, string $key) use ($readableAppID) {
+	array_walk_recursive($spec, function (mixed &$item, string $key) use ($readableAppID) {
 		if ($key == "\$ref" && $item != "#/components/schemas/OCSMeta") {
```

Comment:
is this missing a `is_string($item)` check then?

Because when item is `#/components/schemas/OCSMeta` it's already a string, and otherwise you put the value into `str_replace` which should throw when it's not a string?

Reply:
Hm we could add it, but we can also safely assume that the value of a `$ref` is always a string. If this is not the case then a different part of the code is the problem and not this part.

Reply:
> but we can also safely assume that the value of a $ref is always a string.

Then why is casting it to string a problem?

Reply:
Not all values are strings so we can not cast them, but if the key is `$ref` the value is definitely a string so using `str_replace` is save

Reply:
```suggestion
		if ($key === '$ref' && $item !== '#/components/schemas/OCSMeta') {
```